uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestPresenter.kt

```diff
@@ -50,6 +50,7 @@ class BookingRequestPresenter(view: BookingRequestMVP.View,
     private var outboundTripId: String? = null
     private var quote: Quote? = null
     private var scheduledDate: DateTime? = null
+    private var bookingMeta: HashMap<String, String>? = null
```

Comment:
I suggest renaming this to `metaData`. It's clear that this is for Booking because it's a property of the `BookingRequestPresenter`, but the name is incomplete if we only write "meta". "metaData" is something everyone would understand. It's also consistent with the name set in the UISDKConfiguration `bookingMetadata`.

Reply:
I renamed it to `bookingMetadata` throughout the project, as it is metadata for the booking, it would be easier to understand it. only metadata might give an impression that it is a `metadata` for this component